### PR TITLE
[6378] mako templates 2

### PR DESCRIPTION
**Pull request by efeniks**
**Ticket**: [BUG 6378](https://bugzilla.stxnext.pl/show_bug.cgi?id=6378)
mako templates second version

### CODE REVIEW

File: src/presence_analyzer/views.py [hunk 1]
```diff
@@ -114,5 +115,5 @@ def render_all(temp_name):
     """
     try:
         return render_template(temp_name, selected=temp_name)
-    except TemplateNotFound:
-        return render_template('notfound.html')
+    except mako.exceptions.TopLevelLookupException:
+        return render_template("notfound.html")
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: "?

Comment (mjelonek92) [inline src/presence_analyzer/views.py]: Why this isn't an `abort(404)`?

File: buildout.cfg [hunk 2]
```diff
@@ -25,11 +25,14 @@ logfiles = ${buildout:directory}/var/log
...
 recipe = zc.recipe.egg
-eggs = 
+eggs =
     presence_analyzer
     Paste
     PasteScript
     PasteDeploy
+    Mako
+    Flask-Mako
+    pdb
```
Comment (mjelonek92) [inline buildout.cfg]: Hmm?

File: src/presence_analyzer/templates/base.html [hunk 3]
```diff
@@ -8,32 +8,36 @@
...
-            ] %}
-            <ul>
-                {% for href, caption in menu %}
-                    <li{% if href == selected %} id="selected"{% endif %}>
-                    <a href="{{ href|e }}">{{ caption|e }}</a></li>
-                {% endfor %}
+            ] %>
+
+            % for href, caption in menu:
+                    <li
```
Comment (mjelonek92) [inline src/presence_analyzer/templates/base.html]: Incorrect indentation

**Pull request merged**